Views/Home/LegalNotice.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@{
+    ViewData["Title"] = "Impressum";
+}
+<h1>@ViewData["Title"]</h1>
+<br>
+<p>Angaben gemäß § 5 TMG</p>
+<p><strong>Verantwortungsbereich:</strong><br /> Das Impressum gilt nur f&#xFC;r die Internetpr&#xE4;senz unter der Adresse: <a href="https://schulungen.campus-consult.org">schulungen.campus-consult.org</a></p>
```

Comment:
Instead of `schulungen.campus-consult.org` it should be `personaltool.campus-consult.org` probably, long term it should be dynamic, like for the Schulungsportal, but I guess thats out of scope for this issue/PR